src/bindings/PyDP/algorithms/partition_selection_strategies.cpp

```diff
@@ -0,0 +1,72 @@
+// Provides bindings for Bounded Functions
+
+#include "pybind11/complex.h"
+#include "pybind11/functional.h"
+#include "pybind11/pybind11.h"
+#include "pybind11/stl.h"
+
+#include "algorithms/partition-selection.h"
+
+using namespace std;
+
+namespace py = pybind11;
+namespace dp = differential_privacy;
+
+typedef std::unique_ptr<dp::PartitionSelectionStrategy> PyPartitionSelectionStrategy;
+
+PyPartitionSelectionStrategy create_truncted_geometric_partition_strategy(
+    double epsilon, double delta, int max_partitions_contributed) {
+  dp::PreaggPartitionSelection::Builder builder;
+  builder.SetEpsilon(epsilon);
+  builder.SetDelta(delta);
+  builder.SetMaxPartitionsContributed(max_partitions_contributed);
+
+  auto obj = builder.Build();
+  if (!obj.ok()) {
+    throw std::runtime_error(obj.status().ToString());
+  }
+  return std::move(obj.ValueOrDie());
```

Comment:
Nit (style improvement): no need in `std::move`, the value in `return` is automatically r-value, and automatically will be moved.

This same in other places

Reply:
Apparently - `obj` is of type `base::StatusOr<std::unique_ptr<...>>`, and the method `obj.ValueOrDie()` is overloaded. 
The overloads are:  
```
  const T& ValueOrDie() const&;
  T& ValueOrDie() &;
  const T&& ValueOrDie() const&&;
  T&& ValueOrDie() &&;
```
From [here](https://github.com/google/differential-privacy/blob/114505ef7d70cc25cd5472461d80e51f5ed1c4a8/cc/base/statusor.h#L207-L210). 
The return type is a value type, so the compiler chooses the first version of `const T& ValueOrDie() const&;`, which fails the compilation because `std::unique_ptr` cannot get copied from const ref.